dask/array/core.py

```diff
@@ -3187,6 +3224,16 @@ def cov(m, y=None, rowvar=1, bias=0, ddof=None):
         return (dot(X, X.T.conj()) / fact).squeeze()
 
 
+@wraps(np.corrcoef)
+def corrcoef(x, y=None, rowvar=1):
+    c = cov(x, y, rowvar)
+    if c.shape == ():
+        return c/c
+    d = diag(c)
+    d = d.reshape((d.shape[0], 1))
+    return c / sqrt(d * d.T)
```

Comment:
Copying from an offline conversation, just so others might benefit. My confusion here was that I thought Matt was worried about the computation of the entire `d * d.T` array in memory, which shouldn't happen in either case. Instead, the complaint was that in the current way, the chunks are `n x n`, where in his way the chunks are `1 x n`. Both result in the same shape graphs, with the same dependencies, but one way reduces the number of `n x n` arrays we send between workers.